packages/amplify_authenticator/lib/src/utils/validators.dart

```diff
@@ -112,7 +112,7 @@ FormFieldValidator<String> validatePasswordConfirmation(
         context,
         InputResolverKey.passwordConfirmationEmpty,
       );
-    } else if (getPassword() != passwordConfirmation) {
+    } else if (getPassword().trim() != passwordConfirmation.trim()) {
```

Comment:
Can we trim them when we save them in `authenticator_state.dart` in case this issue comes up elsewhere?

Reply:
Yep, sounds good. Thanks!